src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -33,6 +33,8 @@ public BrowserFixture(IMessageSink diagnosticsMessageSink)
 
         public IMessageSink DiagnosticsMessageSink { get; }
 
+        public string UserProfileDir { get; private set; }
```

Comment:
Is this being set somewhere else?  This is a new property so I was looking for where it gets set in the diff

Reply:
🤧